pkg/controller/logstorage/logstorage_controller_test.go

```diff
@@ -125,6 +126,8 @@ var _ = Describe("LogStorage controller", func() {
 
 		readyFlag = &utils.ReadyFlag{}
 		readyFlag.MarkAsReady()
```

Comment:
If we're going to have two of these, we should rename the other one as well to be more clear.

`readyFlag` -> `tiersReadyflag`